lib/npg_qc/autoqc/role/rna_seqc.pm

```diff
@@ -0,0 +1,87 @@
+#########
+# Author:        Ruben Bautista
+# Created:       2015-08-13
+#
+
+package npg_qc::autoqc::role::rna_seqc;
+
+use Moose::Role;
+use Readonly;
+
+with qw( npg_qc::autoqc::role::result );
+
+our $VERSION = '0';
+
+sub criterion {
+    my $self = shift;
+
+    return q[Fail if RNA-SeQC fails to run];
```

Comment:
Failure to run might have nothign to do with the sequence - not a valid criterium